deploy/helm/sumologic/values.yaml

```diff
@@ -218,6 +218,68 @@ sumologic:
       ## Set provider service (either fluentd or otelcol).
       provider: fluentd
 
+    collector:
+      enabled: false
+
+    multiline:
+      enabled: true
+      start_regex: "^\\[?\\d{4}-\\d{1,2}-\\d{1,2}.\\d{2}:\\d{2}:\\d{2}.*"
```

Comment:
What do you think about the name `first_line_regex` instead of `start_regex`? I feel that "start" is a bit more ambiguous to me.

Also is the `.*` at the end of the regex required?

Reply:
> Also is the .* at the end of the regex required?

it is not afaik

Reply:
> What do you think about the name `first_line_regex` instead of `start_regex`? I feel that "start" is a bit more ambiguous to me.

Sounds good to me, I didn't really put much thought into the name of this option. I don't think there's any standard terminology for this.

Reply:
Done.